pkg/auth/auth_token_handler.go

```diff
@@ -169,3 +170,18 @@ func login(client *ccloudv1.Client, req *ccloudv1.AuthenticateRequest) (*ccloudv
 		return client.Auth.Login(req)
 	}
 }
+
+func (a *AuthTokenHandlerImpl) RevokeRefreshToken(clientFactory CCloudClientFactory, url string, credentials *Credentials) error {
+	req := &ccloudv1.AuthenticateRequest{IdToken: credentials.AuthToken}
+	client := clientFactory.JwtHTTPClientFactory(context.Background(), credentials.AuthToken, url)
```

Comment:
Why can't we just use the default `c.Client` in login/command.go?

Reply:
I mentioned this in slack, but that was the reason I was getting Error 401. I was using the `AnonHTTPClientFactory` before, which I assume is what you mean by default client. However, the backend handler [expects to have a userId in the header](https://github.com/confluentinc/cc-flow-service/blob/b16ae7af553340623a2e3c87daee0728028e34ea/api/v1/handler.go#L168), so using the `JwtHTTPClient` worked.

Reply:
Hmm, seems like this is due to `confluent logout` being an unauthenticated command. Maybe we should switch it to an authenticated command (we should be able to assume that the user is logged in before using it)